src/Superdesk/ContentApiSdk/ContentApiSdk.php

```diff
@@ -0,0 +1,249 @@
+<?php
+
+/**
+ * This file is part of the PHP SDK library for the Superdesk Content API.
+ *
+ * Copyright 2015 Sourcefabric z.u. and contributors.
+ *
+ * For the full copyright and license information, please see the
+ * AUTHORS and LICENSE files distributed with this source code.
+ *
+ * @copyright 2015 Sourcefabric z.ú.
+ * @license http://www.superdesk.org/license
+ */
+
+namespace Superdesk\ContentApiSdk;
+
+use Superdesk\ContentApiSdk\Client\ClientInterface;
+use Superdesk\ContentApiSdk\Data\Item;
+use Superdesk\ContentApiSdk\Data\Package;
+use Superdesk\ContentApiSdk\Exception\InvalidDataException;
+use stdClass;
+
+/**
+ * Superdesk ContentApi class.
+ */
+class ContentApiSdk
+{
+    const SUPERDESK_ENDPOINT_ITEMS = '/items';
+    const SUPERDESK_ENDPOINT_PACKAGES = '/packages';
+
+    /**
+     * Any (http) client that implements ClientInterface.
+     *
+     * @var ClientInterface
+     */
+    protected $client;
+
+    /**
+     * Construct method for class.
+     *
+     * @param ClientInterface $client
+     */
+    public function __construct(ClientInterface $client)
+    {
+        $this->client = $client;
+    }
+
+    /**
+     * Get a single item via id.
+     *
+     * @param  string $itemId Identifier for item
+     *
+     * @return Item
+     */
+    public function getItem($itemId)
+    {
+        $body = $this->client->makeApiCall(sprintf('%s/%s', self::SUPERDESK_ENDPOINT_ITEMS, $itemId));
+
+        $jsonData = json_decode($body);
+        if (is_null($jsonData) || json_last_error() !== JSON_ERROR_NONE) {
+            throw new InvalidDataException('Returned data is not in (valid) json format.', json_last_error());
+        }
+
+        $item = new Item($jsonData);
+
+        return $item;
+    }
+
+    /**
+     * Get multiple items based on a filter.
+     *
+     * @param  array $params Filter parameters
+     *
+     * @return mixed
+     */
+    public function getItems($params)
+    {
+        $return = null;
+        $body = $this->client->makeApiCall(self::SUPERDESK_ENDPOINT_ITEMS, $params);
+
+        $bodyJSONObj = json_decode($body);
+        if (is_null($bodyJSONObj) || json_last_error() !== JSON_ERROR_NONE) {
```

Comment:
those two lines can be extracted to private static method `::getValidJsonObj($body)` with object|false response